app/controllers/user_controller.rb

```diff
@@ -13,31 +14,38 @@ def signup
       render json: @new_user, status: 201
     else
       render json: @new_user.errors.details, status: 500
-
     end
   end
 
   def update_user
     current_user = User.find(params[:id])
     payload = JSON.parse(request.body.read)
 
-    current_user.first_name = payload['first_name']
-    current_user.email = payload['email']
-    current_user.user_type = payload['last_name']
-    current_user.user_type = payload['password']
-    current_user.user_type = payload['user_type']
-    
-    
+    current_user.first_name = payload["first_name"]
+    current_user.email = payload["email"]
+    current_user.user_type = payload["last_name"]
+    current_user.user_type = payload["password"]
+    current_user.user_type = payload["user_type"]
+
```

Comment:
I think you have a mistake here, you are updating the user_type multiple times with different values.

Reply:
Also, I'm not sure this approach is the best because if there's no last_name change request in the payload, it will update the users' last name to nil.
I think there could be a way to user active records' update method that updates only the fields passed as a hash.

Reply:
Hello, @Opio-Emmanuel-Omona. Thanks for the feedback. Please note that my PR is on show method not update method. The changes you request is to be treated as a separate bug story.

Reply:
I see, so it looks like you just changed the formatting.
If that's the case, at least change the user_type duplication in this PR, as for the implementation change it can be in another story.

Reply:
Either way, remember to add a test for `last_name` and `password` when you make the change.

Reply:
Alright. A story has been created on this method to be treated as an independent endpoint. This is to maintain the workflow. Thanks